Okta/OktaError.swift

```diff
@@ -68,6 +69,8 @@ extension OktaError: LocalizedError {
             return NSLocalizedString("Error finding the user info endpoint.", comment: "")
         case .ParseFailure:
             return NSLocalizedString("Failed to parse and/or convert object.", comment: "")
+        case .MissingUserIdToken:
+            return NSLocalizedString("User ID token needed to fulfill this operation.", comment: "")
```

Comment:
We can probably remove "User" for the error name and message, as an ID token implies that it is bound to a specific user.

`MissingUserIdToken` -> `MissingIdToken`
`"User ID token needed ..."` -> `"ID token needed ..."`